apprise/plugins/NotifyPushover.py

```diff
@@ -288,9 +288,10 @@ def __init__(self, user_key, token, targets=None, priority=None,
         self.sound = NotifyPushover.default_pushover_sound \
             if not isinstance(sound, str) else sound.lower()
         if self.sound and self.sound not in PUSHOVER_SOUNDS:
-            msg = 'The sound specified ({}) is invalid.'.format(sound)
-            self.logger.warning(msg)
-            raise TypeError(msg)
+            msg = 'The sound specified ({}) is not a default sound. ' \
+                  'Ensure you have a custom sound with this name defined.' \
+                  .format(sound)
+            self.logger.info(msg)
```

Comment:
I think it would be worth just dropping all of this if we're allowing free-form sounds (no sense generating a log entry).  I wonder if we can just dump all of the PUSHOVER_SOUNDS object entirely if we don't' really need to look at it anymore?

Thoughts?

Reply:
That makes sense. I haven't had time to look back at this, but since my birdnet-pi install is pulling in apprise for notifications, and I'd like to use the custom sound support within it, I'll try and rework this PR soon. Thanks!

Reply:
I re-reviewed what you've done; i only added one small change (switching `info` to `debug`). I'll go ahead and merge what you've added.